stroeer/core/v1/article.proto

```diff
@@ -940,155 +940,162 @@ message Asset {
 
 
 /**
-* @FileArticle Body
+* @FileArticle Content
 */
 
 /**
 * ```protobuf
-* message Body {}
+* message Content {}
 * ```
-* Body of the article to render on detail pages.
+*
 */
-message Body {
+message Content {
```

Comment:
If everyone agreed on this naming, I will add more docs here

Reply:
This is fine 👍